### DISCUSSION TITLE
Implementing a Centered Marker Location Picker like Uber and Grab in React Native using Mapbox

### DISCUSSION THREAD

**Discussion by miketrinh1995** [rnmapbox/maps#3690] [Q&A]
I am trying to implement a location picker functionality in React Native using Mapbox, similar to that of Uber or Grab, where the selected location is always at the center of the screen. 

However, unlike the typical behavior, I want the selected location to remain unchanged even when zooming in or out, and only update when the map is dragged to a new location. 

I found an example implementing this for the web [Link code here,](https://jsfiddle.net/hwv0e6nr/21/) but I can't find matching properties or methods in the Mapbox SDK for React Native.

**Comment by 6ewis** [comment]
thumbs up.

**Comment by joseuride** [comment]
Same here, we really need it.
